### PR TITLE
TAO-1815_Record_response_type

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

File: views/js/qtiXmlRenderer/renderers/ResponseProcessing.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ define([
     'tpl!taoQtiItem/qtiXmlRenderer/tpl/responseProcessing',
     'tpl!taoQtiItem/qtiXmlRenderer/tpl/responses/match_correct',
     'tpl!taoQtiItem/qtiXmlRenderer/tpl/responses/map_response',
-    'tpl!taoQtiItem/qtiXmlRenderer/tpl/responses/map_response_point'
+    'tpl!taoQtiItem/qtiXmlRenderer/tpl/responses/map_response_point',
```
Comment (ssipasseuth) [inline views/js/qtiXmlRenderer/renderers/ResponseProcessing.js]: trailing comma

Comment (hutnikau) [inline views/js/qtiXmlRenderer/renderers/ResponseProcessing.js]: removed

File: views/js/qtiXmlRenderer/tpl/responseDeclaration.tpl [hunk 2]
```diff
@@ -11,7 +11,11 @@
     {{~#if correctResponse.length}}
     <correctResponse>
         {{~#each correctResponse}}
+        {{~#if ../isRecord}}
+        <value fieldIdentifier="{{fieldIdentifier}}" baseType="{{baseType}}"><![CDATA[{{{value}}}]]></value>
```
Comment (ssipasseuth) [inline views/js/qtiXmlRenderer/tpl/responseDeclaration.tpl]: A Record may be recursive. So I believe we need to manage the case of having a recursive object.

Comment (ssipasseuth) [inline views/js/qtiXmlRenderer/tpl/responseDeclaration.tpl]: Ok, i seems that QTI xsd does not allow recursive record, so let's forget it.

**Pull request merged**